examples/java/hello/pom.xml

```diff
@@ -11,7 +11,12 @@
     <dependency>
       <groupId>org.cdk8s</groupId>
       <artifactId>cdk8s</artifactId>
-      <version>0.25.0</version>
+      <version>0.26.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.cdk8s</groupId>
+      <artifactId>cdk8s-plus</artifactId>
```

Comment:
I don't think we should add `cdk8s-plus` to an example that doesn't actually need it. Its good to have it as part of the template as an easy kick-off, but working examples can be left as is.

Reply:
Done